### PR TITLE
16 seed sql db using data generated from fakerjs

### PR DESCRIPTION
**Pull request by polarbear23**

### CODE REVIEW

File: src/server/utils/faker.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const password = faker.internet.password();
+    const email = faker.internet.email(username);
+
+    return { username, password, email }
+}
+
+const createFakeCompetition = () => {
+    const title = faker.lorem.sentence();
+
+    return { title, }
```
Comment (bravint) [inline src/server/utils/faker.js:14]: remove comma pls

File: prisma/seed.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const { createFakeAdmin, createFakeCompetition, createFakeCompetitors, createFakeSeasons, createFakeTeam, createFakeRound, createFakePlacement, createFakePositionMapping } = require('../src/server/utils/faker');
```
Comment (bravint) [inline prisma/seed.js:3]: Prettier should have split this line up? That is one gigantic line

**Review by bravint — APPROVED** [review]

File: src/server/utils/faker.js (hunk 1 above)
Comment (polarbear23) [reply src/server/utils/faker.js:14]: ok

File: prisma/seed.js (hunk 2 above)
Comment (polarbear23) [reply prisma/seed.js:3]: ok I fix

**Comment by nathanSamosa** [comment]
great job!

**Review by nathanSamosa — APPROVED** [review]

**Review by molloyconor511 — APPROVED** [review]

File: prisma/seed.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    }
+    return rounds;
+
+}
+
+const createPlacements = async (participants, rounds) => {
+    const placements = [];
+
+    for (let i = 0; i < rounds.length; i++) {
+        const positions = Array.from({ length: participants.length }, (_, i) => i + 1); //fills an array up to max of participants [1,2,3 ... participants.length]
```
Comment (vherus) [inline prisma/seed.js:137]: Great commenting

File: prisma/seed.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+            positions.splice(index, 1);
+            console.log("placement", placement);
+            placements.push(placement);
+        }
+    }
+    return placements
+}
+
+const createPositionMappings = async (season, participants) => {
+    const positions = Array.from({ length: participants.length }, (_, i) => i + 1); //fills an array up to max of participants [1,2,3 ... participants.length]
```
Comment (vherus) [inline prisma/seed.js:166]: I've seen this same line twice now, good candidate to be moved into its own function!

File: prisma/seed.js [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+        positionMappings.push(positionMapping);
+    }
+    return positionMappings
+}
+
+
+const seed = async () => {
+    const admins = await createAdmin();
+    for (let i = 0; i < admins.length; i++) {
+        for (let j = 0; j < admins[i].competition[0].seasons.length; j++) {
```
Comment (vherus) [inline prisma/seed.js:191]: Make a note of this, it'll be good to revisit what you've done here after you've been through the computer science week - specifically time complexity / "Big O notation"

**Review by vherus — APPROVED** [review]
Nice work, some great demonstration of your understanding in this PR.

**Pull request merged**